client/signup/steps/woocommerce-install/confirm/index.tsx

```diff
@@ -156,7 +149,7 @@ export default function Confirm( props: WooCommerceInstallProps ): ReactElement
 					<ActionSection>
 						<SupportLink />
 						<StyledNextButton
-							disabled={ hasBlockers || ! isDataReady || isAtomicSite }
+							disabled={ hasBlockers || ! isDataReady }
```

Comment:
This might need to be reverted, I don't think we display this when the site is already atomic though so maybe not.